src/aten/sycl/Math.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <ATen/AccumulateType.h>
+#include <ATen/NumericUtils.h>
+#include <c10/macros/Macros.h>
+#include <c10/util/MathConstants.h>
+#include <c10/util/complex.h>
+
+namespace at {
+namespace native {
+
+template <typename scalar_t>
+static inline scalar_t calc_gcd(scalar_t a_in, scalar_t b_in) {
```

Comment:
Remove it.

Reply:
Reuse implementation in ATen/native/Math.h. Arithmetic of basic data type should be common and shared with different backends.